packages/frontend/tsconfig.build.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["src/view"],
+  "include": ["src"],
```

Comment:
Why is this change required?

Reply:
yarn build was not catching obvious syntax errors in tsx files.